mobile/src/components/Report.js

```diff
@@ -24,6 +24,7 @@ const { lightThemeText } = commonStyles;
 const fontWidth = 8;
 const rowHeight = 22;
 const sortIconSize = 16;
+const maxPageSize = 2147483647;
```

Comment:
Can you use Number.MAX_SAFE_INTEGER instead?

Reply:
it's value is 9007199254740991  so it wouldn't be suitable here

Reply:
I see, this is max integer on backend side.